sqlalchemy_redshift/dialect.py

```diff
@@ -135,6 +154,24 @@ class RedshiftImpl(postgresql.PostgresqlImpl):
 ])
 
 
+class TIMESTAMPTZ(sa.dialects.postgresql.TIMESTAMP):
+    """
+    Redshift defines a TIMTESTAMPTZ column type as an alias
+    of TIMESTAMP WITH TIME ZONE.
+    https://docs.aws.amazon.com/redshift/latest/dg/c_Supported_data_types.html
+
+    Adding an explicit type to the RedshiftDialect allows us follow the
+    SqlAlchemy conventions for "vendor-specific types."
+
+    https://docs.sqlalchemy.org/en/13/core/type_basics.html#vendor-specific-types
+    """
```

Comment:
```suggestion
    """ # noqa
```

Adding a noqa comment here should satisfy the linter.